src/components/Dashboard/TodoList.vue

```diff
@@ -1498,9 +1568,9 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
   async loadAgmExtension (task: ApiTaskIF): Promise<void> {
     const filing = task.task.filing
     const header = filing.header
-    const agmLocationChange = filing.agmLocationChange
+    const agmExtension = filing.agmExtension
```

Comment:
@JazzarKarim This is a bugfix. It might be what Riyaz is talking about [here](https://teams.microsoft.com/l/message/19:c447ed74b25b49d4b59cf0302037baab@thread.tacv2/1700770751545?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=00bd6377-9feb-44be-bbc8-19c3752a67b2&parentMessageId=1700770751545&teamName=External%3A%20SBC%20REG%20Modernization&channelName=Entities%20Product%20Team&createdTime=1700770751545). Can  you follow up with Riyaz, please?

Reply:
Sure thing Sev. I'll have a look.

Reply:
This fixes the bug Riyaz mentioned Sev: https://business-filings-dev--pr-590-9g6gr3iv.web.app/BC0871367